Source/ARTLocalDevice.m

```diff
@@ -102,6 +95,33 @@ + (ARTLocalDevice *)load:(NSString *)clientId storage:(id<ARTDeviceStorage>)stor
     return device;
 }
 
+- (BOOL)setupDetailsWithClientId:(NSString *)clientId {
```

Comment:
What's the return value of this method for? Nobody uses it.

Reply:
See response [here](https://github.com/ably/ably-cocoa/pull/1847/files#r1442138958)

Reply:
https://github.com/ably/ably-cocoa/pull/1847/commits/c64c56192644dfc2a865c2b41a411be72d2bfb71